multinode/bootstrapping.md

```diff
@@ -0,0 +1,190 @@
+current database on the access node. For example, if you are connected
+to the `postgres` database on the access node, the database name will
+be `postgres` on the data node as well.
+
+To provision the data node in a different database on the instance use
+the `database` option to the [`add_data_node`][add_data_node] command,
+for example:
+
+```
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com',
+                            database => 'my_database');
+```
+
+This is particularly useful if you want to experiment with having data
+nodes and access nodes in the same Postgres instance.
+
+### Bootstrapping the database
+
+If `bootstrap` is true (which is the default), the access node will
+first connect to either the `postgres` database or, if the `postgres`
+database does not exist, to the `template0` database on the data node,
+and create a new database:
+
+```
+CREATE DATABASE <i>database</i>
+   TEMPLATE template0 OWNER <i>owner</i>
+   ENCODING <encoding> LC_COLLATE <i>collation</i>
+   LC_CTYPE <i>character type</i>
+```
+
+Here, the `owner` is the user connected on the access node and the
+`encoding`, `collation`, and `character type` parameters are taken
+from the current database on the access node.
+
+Once the database is created, the access node disconnects and continues
+with bootstrapping the extension. If the database already exists on
+the data node, the procedure validates the database and proceeds with
+setting up the extension.
+
+### Validating the database
+
+The database will be validated by checking that the encoding,
+character type, and collation are the same as the current database on
+the access node.
+
+## Setting up the extension
+
+The `timescaledb` extension needs to be installed in the database on
+the data node, so the next step is ensuring that there is a valid
+extension installed in the database by either:
+
+* Creating the extension.
+* Verifying that the extension is compatible with the extension
+  installed on the access node.
+
+### Creating the schema for the extension
+
+To install the extension, a schema is created to store the objects
+that the `timescaledb` extension needs. The schema name is decided by
+looking at what schema name is used for the extension on the access node and
+using the same name on the data node. If you have created the
+`timescaledb` extension in the default manner, `public` will be used
+for the schema on the access node, hence `public` will be used on the
+data node as well.
+
+If the schema name for the extension on the access node is `public`,
+no new schema is created, but otherwise it is expected that the schema
+name does *not* exist. In the event that the schema name exists (but
+is not `public`), an error will be raised and the bootstrapping will abort.
+
+To change the schema used on the access node and data node, you can
+create a new schema, then create the extension with the schema that
```

Comment:
```suggestion
create a new schema on the access node, then create the extension with the schema that
```
It is easy to confuse otherwise with the data node.